### PR TITLE
[Merged by Bors] - Update rodio 0.15

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- Update rodio to 0.15
- Replace #3828

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: deny.toml [hunk 1]
```diff
@@ -43,22 +42,12 @@ skip = [
...
-    { name = "ndk-macro", version = "0.2" },            # from rodio v0.14.0
-    { name = "ndk-glue", version = "0.3" },             # from rodio v0.14.0
-    { name = "ndk-glue", version = "0.5" },             # from rodio v0.14.0
-    { name = "ndk-sys", version = "0.2" },              # from rodio v0.14.0
-    { name = "nix", version = "0.20" },                 # from rodio v0.14.0
-    { name = "proc-macro-crate", version = "0.1" },     # from rodio v0.14.0
+    { name = "ndk", version = "0.5" },                  # from winit v0.26.1
+    { name = "ndk-glue", version = "0.5" },             # from winit v0.26.1
+    { name = "ndk-sys", version = "0.2" },              # from winit v0.26.1
     { name = "stdweb", version = "0.1" },               # from rodio v0.14.0
```
Comment (hymm) [inline deny.toml:50]: > ` # from rodio v0.14.0`

does this line need to be updated too?

**Comment by alice-i-cecile** [comment]
Supports #3772.

File: deny.toml (hunk 1 above)
Comment (mockersf) [reply deny.toml]: yup, thanks!

**Review by IceSentry — APPROVED** [review]

**Pull request closed without merging**